src/ui/PreferencesScreen.cpp

```diff
@@ -468,5 +475,9 @@ lif::ui::Action PreferencesScreen::_changeNPlayers(int newNPlayer) {
 
 void PreferencesScreen::onUnload() {
 	lif::ui::Screen::onUnload();
-	lif::savePreferences(lif::PREFERENCES_SAVE_FILE_NAME);
+	#if defined(SFML_SYSTEM_MACOS)
+		lif::savePreferences(std::string(lif::saveDir + std::string(lif::PREFERENCES_SAVE_FILE_NAME)).c_str());
```

Comment:
Since this is done in a couple of places maybe it's better to either have a `lif::getPreferencesPath()` function or building a `lif::preferencesPath` alongside `lif::assetDir` / `lif::saveDir`

Reply:
Will do